### PR TITLE
Use case fixes

### PR DESCRIPTION
**Pull request by tinashechiraya**

### CODE REVIEW

**Comment by tinashechiraya** [comment]
email_exists
email_invalid
notification_on_tab

**Comment by tinashechiraya** [comment]
content_fills
notifications
now content fills on the page , compared to the current on staging site

File: django_project/stakeholder/views.py [hunk 1]
```diff
@@ -28,10 +28,23 @@
 from core.celery import app
 from frontend.views.base_view import RegisteredOrganisationBaseView
 from frontend.serializers.stakeholder import ReminderSerializer
+from django.contrib.auth.models import User
 logger = logging.getLogger(__name__)
 
 
 
+def check_email_exists(request):
```
Comment (danangmassandy) [inline django_project/stakeholder/views.py:36]: @tinashechiraya can this API be accessed by public or only by registered user? if this is for registered user, then please protect it by using IsAuthenticated from rest_framework.

**Review by danangmassandy — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 13 min

**Review by danangmassandy — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 15 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 13 min

**Pull request merged**